integration/tests/access/consensus_follower_test.go

```diff
@@ -110,17 +118,94 @@ func (suite *ConsensusFollowerSuite) TestReceiveBlocks() {
 			}
 		}
 		// wait for finalized blocks
-		unittest.AssertReturnsBefore(suite.T(), receiveBlocks, 2*time.Minute) // waiting 2 minute for the missing 5 blocks
+		unittest.AssertReturnsBefore(s.T(), receiveBlocks, 2*time.Minute) // waiting 2 minute for the missing 5 blocks
 	})
 }
 
-func (suite *ConsensusFollowerSuite) buildNetworkConfig() {
+// TestSignerIndicesDecoding tests that access node uses signer indices' decoder to correctly parse encoded data in blocks.
+// This test receives blocks from consensus follower and then requests same blocks from access API and checks if returned data
+// matches.
+func (s *ConsensusFollowerSuite) TestSignerIndicesDecoding() {
```

Comment:
I think this test should go in `AccessSuite` since it's not testing any functionality of the consensus follower.